sdk/include/aditof/camera.h

```diff
@@ -279,6 +279,20 @@ class SDK_API Camera {
      * @return Status
      */
     virtual Status adsd3500GetRadialThresholdMax(int &threshold) = 0;
+
+    /**
+     * @brief Get the sensor temperature threshold
+     * @param[out] temperature
+     * @return Status
+     */
+    virtual Status adsd3500GetSensorTemperature(uint16_t &tmpValue) = 0;
+
+    /**
+     * @brief Get the laser temperature threshold
+     * @param[out] temperature
```

Comment:
Do we know the measurement unit? If yes, can you mention it here in the description of the function?

Reply:
added measurement unit